src/html/blocks/clients.html

```diff
@@ -0,0 +1,29 @@
+<div class="clients">
+    <div class="clients__logo">
+        <div class="clients__img-wrap">
+            <img alt="Deorham networks'" src="/images/clients/client-1.png" />
+        </div>
+        <div class="clients__img-wrap">
+            <img alt="Ratings" src="/images/clients/client-2.png" />
+        </div>
+        <div class="clients__img-wrap">
+            <img alt="Malic Media" src="/images/clients/client-3.png" />
+        </div>
+        <div class="clients__img-wrap">
+            <img alt="Because." src="/images/clients/client-4.png" />
+        </div>
+        <div class="clients__img-wrap">
+            <img alt="Womgily" src="/images/clients/client-5.png" />
+        </div>
+    </div>
```

Comment:
Вот так достаточно сделать:

```html
<div class="clients__img-wrap">
    <img class="clients__img" alt="Deorham networks'" src="./images/clients/client-1.png" />
    <img class="clients__img" alt="Ratings" src="./images/clients/client-2.png" />
    <img class="clients__img" alt="Malic Media" src="./images/clients/client-3.png" />
    <img class="clients__img" alt="Because." src="./images/clients/client-4.png" />
    <img class="clients__img" alt="Womgily" src="./images/clients/client-5.png" />
</div>
```